example/centos/centos-9-aarch64-edge-commit.yaml

```diff
@@ -1,6 +1,7 @@
 otk.version: "1"
 
 otk.define:
+  architecture: aarch64
```

Comment:
This is really interesting, the `edge-commit` between the two arches is now fully identical except for the package list